client.go

```diff
@@ -160,7 +160,7 @@ type APIClient interface {
 	GetKeys(keys map[string]string) ([]string, error)
 	DeleteKey(keyID string) error
 	GetACL(keyID string) (*ACL, error)
-	PutAccess(keyID string, a *Access) error
+	PutAccess(keyID string, acl ...Access) error
```

Comment:
Do we have a test to make sure that old knox client binaries won't break?

Reply:
yes there is a server side test in routes_test.go to ensure that the old apis still work with the access parameter. I'll add a test for the new acl parameter there as well (so it is clear that both are tested).

I also manually tested with dev_client and dev_server and the old and new clients both work.